### ISSUE TITLE
Raw Email Data Output

### ISSUE THREAD

**Issue by kLOsk** [kodie/gravityforms-repeater#10]
Hi Kodie,
I seem to have encountered another issue with the plugin :-/

I am trying to generate a Raw Text Email, but it seems the plugin does not support this. Here is my Notification code, with Autoformatting disabled:

```
xxx {[Field Name]:[field_id]}
Single Line: {Single Line:1}
{Repeater:6}
Paragraph: {Paragraph:2}
Radio: {Radio:3}
{Untitled:7}
List: {List:4:}
Col: {Column List:5:}
File: {File:8}
```

I'm using a filter to generate text emails:

```
// Set Gravity Forms to text Emails
add_filter( 'gform_notification', 'change_notification_format', 10, 3 );
function change_notification_format( $notification, $form, $entry ) {

    // change notification format to text from the default html
    $notification['message_format'] = 'text';

    return $notification;
}
```

Emails arrive like this: 

```
xxx {[Field Name]:[field_id]}
Single Line: dsaad
a:2:{i:1;a:2:{i:2;a:1:{i:0;s:5:"sddsa";}i:3;a:1:{i:0;s:5:"Other";}}i:2;a:2:{i:2;a:1:     {i:0;s:13:"dasdsaasdsads";}i:3;a:1:{i:0;s:5:"other";}}}
 Paragraph:  
Radio: 
```

List: sdads
Col: sad,ads,dsa
File:

**Comment by kodie** [comment]
Hey! I will look into this today and get back to you!

**Comment by kodie** [comment]
@kLOsk I fixed this in the 8c3a5da commit. Test it out using the latest development version of the plugin.

See how to get the development version of the plugin here:
https://github.com/kodie/gravityforms-repeater#development
